.kitchen.yml

```diff
@@ -0,0 +1,17 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+
+platforms:
+  - name: ubuntu-12.04
+  - name: ubuntu-14.04
+  - name: centos-6.5
```

Comment:
I'll be breaking this PR up into more managable chunks, so I'll address test-kitchen with the latest boxes in a new PR.